### ISSUE TITLE
GraphQL Playground does not work offline

### ISSUE THREAD

**Issue by pavellishin** [prisma/graphql-playground#1069]
#### This issue pertains to the following package(s):

- [ ] GraphQL Playground - Electron App
- [x] GraphQL Playground HTML
- [x] GraphQL Playground
- [ ] GraphQL Playground Express Middleware
- [ ] GraphQL Playground Hapi Middleware
- [ ] GraphQL Playground Koa Middleware
- [ ] GraphQL Playground Lambda Middleware

#### What OS and OS version are you experiencing the issue(s) on?

OS X 10.13.6

#### What version of graphql-playground(-electron/-middleware) are you experiencing the issue(s) on?

1.7.26

#### What is the expected behavior?

When I am offline, I expect the local GraphQL Playground to load and work against the local server.

#### What is the actual behavior?

The playground does not load, and displays some CDN-related errors.

#### What steps may we take to reproduce the behavior?

Disconnect from the internet, and try to load a local GraphQL playground instance.

_Please provide a gif or image of the issue for a quicker response/fix._

![image](https://user-images.githubusercontent.com/296084/63176577-2ee2b880-c014-11e9-8e5e-4f8d0d02f0fa.png)

**Comment by acrogenesis** [comment]
@yoshiakis were you able to solve this problem?

**Comment by acrogenesis** [comment]
@pavellishin were you able to solve this problem?

**Comment by pavellishin** [comment]
No, but I haven't been actively trying, and I haven't been in an offline-work-situation since then.

**Comment by visortelle** [comment]
I faced the same issue today.

Is there any sense of using CDN here? Sure, it should be fixed.

**Comment by deadcoder0904** [comment]
Me too, faced the same issue yesterday when my net went off & I wasn’t able to open Playground.

Have no idea why a CDN is used, instead of a local build.

Would love to see this fixed 👍

**Comment by science4reason** [comment]
This is also an issue for Strapi users where this playground is a plugin.

**Comment by egonzal93** [comment]
In my case the cdn is being blocked by the company so I cannot use the playground. There should be an option to not use the cdn here.

**Comment by d4g0** [comment]
I see im not the first  and surely not the last in have the same issue with the cnd bundles. I know in a ideal world all we developers  will have nice and fast internet connections, but thias far far away for been a reallity for everyone, for instance i live in cuba, an i have the "privilege" of have a time messured, expensive and really slown internet service. It will be nice to bundle the `react` and `css` files with the repo and leave a note or indication of how to use those instead. As i do my self downloading the cdn bundles and deployit as static with the same  express server, and offcourse 'tuning' the original html assamble section of the playground code ( lines 15 and 19 of the `graphql-playground-html/dist/render-playground-page.js` ).

**Comment by pavellishin** [comment]
I don't think they'll get around to this issue, y'all, given that they announced that this is going bye-bye way back in April :/ https://github.com/graphql/graphql-playground/issues/1143
